README.md

```diff
@@ -561,6 +561,59 @@ Use it to type in your key and install.
 
 Go back to Step 4A-4 and then finish with Step 4A-5.
 
+---
+## Forks
+
+### Which Fork Should I Use?
+
+If you have to ask, you're not ready for forks.
```

Comment:
Maybe the caution should be moved up and this should end with "Here are the reasons why:"

Reply:
Yeah it could use a cleanup. I think it's messy because chrispypatt is transcribing what I wrote in the past. Hey Chris, go wild and overhaul it in your own voice.

Reply:
I like all the points you had @calvinpark. I will move the caution up and will think for a bit on if there is a different way I want to organize these points or if there is anything I want to add.